conf/gis.config

```diff
@@ -0,0 +1,27 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for GIS (Aquila)
+ * -------------------------------------------------
+ * Defines reference genomes, using iGenome paths
+ * Imported under the default 'standard' Nextflow
+ * profile in nextflow.config
+ */
+
+process {
+  executor = 'sge'
+  clusterOptions = { "-l mem_free=" + task.memory.toString().replaceAll(/[\sB]/,'') }
+  penv = 'OpenMP'
+  maxRetries = 3
+  errorStrategy = { task.attempt < 2 ? 'retry' : 'finish' }
```

Comment:
How is this different to just setting `maxRetries` to `1`?

Reply:
We use this internally.

Reply:
Ok. I think it may be cleaner / clearer code to just use `maxRetries = 1` and delete this line if you don't want the pipeline to rerun failed tasks. Or in fact just `errorStrategy = 'finish'` and delete the `maxRetries` line. Both should have the same behaviour.